niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/InstallPermissionTester.java

```diff
@@ -177,35 +177,52 @@ public InstallPermissionTester(TestConfiguration configuration, Activity activit
                 new PermissionTest(false, () -> mConnectivityManager.getActiveNetwork()));
 
         mPermissionTasks.put(ACCESS_WIFI_STATE,
-                new PermissionTest(false, () -> mWifiManager.getConfiguredNetworks()));
+                new PermissionTest(false, () -> {
+                    if (ActivityCompat.checkSelfPermission(mContext, Manifest.permission.ACCESS_FINE_LOCATION)
+                            != PackageManager.PERMISSION_GRANTED) {
+                        throw new BypassTestException(
+                                "ACCESS_FINLE_LOCATION permission should be granted to run this test case");
+                    }
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+                        //mWifiManager.getCallerConfiguredNetworks();
```

Comment:
Did you intend to use this method for devices running S+? Or can the device SDK version check be skipped and `mWifiManager.getConfiguredNetworks` called for all device SDK versions?

Reply:
getConfiguredNetworks api is obsolated after SDK31. I will check the behaviour of this test again...

Reply:
sorry it's my mistake in this case we should call getConfiguredNetworks in any version as of now.
in future, the api would be osolated, but not yet.

Reply:
sent fix

Reply:
Update looks good; thanks!